<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Invaders</class>
 <widget class="QWidget" name="Invaders">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invaders</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Invaders {background-color: rgb(0, 0, 0);}</string>
  </property>
  <widget class="QWidget" name="gameWidget" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>800</width>
     <height>1100</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="leaderBoardWidget" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>782</width>
     <height>323</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="menuFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>881</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Small Semibol</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QLabel" name="menuBgAnimContainer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1071</width>
      <height>428</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="itemsFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>881</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainMenuLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="gameTitle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/resources/ui/spaceInvasion.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
